packages/core/test/globals.js

```diff
@@ -0,0 +1,41 @@
+const test = require('ava')
+const makeGetEndowmentsForConfig = require('../src/makeGetEndowmentsForConfig.js')
+const {
+  createScenarioFromScaffold,
+  runScenario
+} = require('./util')
+
+test('globals - ensure window.document getter behavior support', async (t) => {
```

Comment:
lets copy this test a make a more direct copy of the `window.document` behavior where we use the same names (`globalThis.document`) it throws if the this-value is not as expected